src/processors/data_processor.py

```
@@ -0,0 +1,640 @@
+            data_lines = lines[data_start_idx:]
+            with output_file.open("w", encoding="utf-8") as f:
+                f.writelines(data_lines)
+
+            logger.info(f"全数報告処理成功: {source_file.name} → {output_filename}")
+
+            # ログ記録
+            self._log_processing(source_file, [output_file], metadata)
+
+            return NormalizationResult(success=True, source_path=source_file, output_files=[output_file])
+
+        except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)